Assets/Scripts/Advertisements/AdvertisementInitializer.cs

```diff
@@ -0,0 +1,47 @@
+using UnityEngine.Advertisements;
+using UnityEngine;
+
+namespace UntitledCube.Advertisements
+{
+    public class AdvertisementInitializer : MonoBehaviour, IUnityAdsInitializationListener
+    {
+        [SerializeField] bool _testMode = true;
+        
+        private const string ANDROID_GAME_ID = "5562758";
+        private const string IOS_GAME_ID = "5562759";
```

Comment:
Do we need this?

Reply:
Same for the rest of the script do we need IOS Support?

Reply:
API needs the ios and android key, and is always nice for if we port it to ios